Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionComposite.shader

```diff
@@ -17,7 +17,7 @@
         "Stencil" :
         {
             "Enable" : true,
-            "ReadMask" : "0x7F",
+            "ReadMask" : "0x3F",
```

Comment:
I am assuming you tested the reflection features to ensure they are still working.

Reply:
I *think* I did but it's been a minute since I created the feature and it's possible dev has changed since I did.  Another round of testing couldn't hurt.

Reply:
Full screen reflections seem to work OK
![image](https://github.com/o3de/o3de/assets/26804013/6f461b17-a8e0-4f49-a664-b19cc5aff045)
But I am noticing that when ray tracing is enabled the "Never" silhouette is drawn in the reflection:
![image](https://github.com/o3de/o3de/assets/26804013/d028786d-33d8-4574-93ee-6555faff9255)
But, I'm fine with filing that as a known issue for now.

Reply:
OK I tested with multiple and nested reflection probes and captured in RenderDoc to verify the stencil masks looked correct 
![image](https://github.com/o3de/o3de/assets/26804013/240acd35-e494-4639-831b-e4dfe8575a4b)

Reply:
![image](https://github.com/o3de/o3de/assets/26804013/1244af53-f9bf-4ea1-9353-41ab3b261a10)

This shows the stencil buffer from a previous test with nested reflection probes (dark grey shades) and one silhouette blocker (the bright white part)